### PR TITLE
Python 3

### PR DESCRIPTION
**Pull request by daniellindau**
Update the example to Python 3

This makes it python 3 only, no more 2.7. I suggest we tag master before merging this, so that you can use that version if you really cant use python 3.

### CODE REVIEW

File: settings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "client_id": "client-1",
   "client_secret": "Password1",
-  "audience": "client-one",
+  "audience": "client-1",
```
Comment (travisspencer) [inline settings.json]: why this?

File: client.py [hunk 2]
```diff
@@ -93,18 +91,22 @@ def get_authn_req_url(self, session, acr, forceAuthN, scope):
         """
         state = tools.generate_random_string()
         session['state'] = state
-        session['code_verifier'] = code_verifier = tools.generate_random_string(100)
+        session['code_verifier'] = code_verifier = tools.generate_random_string(100).encode()
 
-        code_challenge = tools.base64_urlencode(hashlib.sha256(code_verifier).digest())
+        code_challenge = self.__create_code_challenge(code_verifier)
```
Comment (travisspencer) [inline client.py]: hard to tell; where is this method?

Comment (daniellindau) [reply client.py]: It's on line 106. Its the above line but moved to a function. I had more changes in there before, but the result became the same thing. So I could revert it, but it also reads a little bit easier like this I think.

File: settings.json (hunk 1 above)
Comment (daniellindau) [reply settings.json]: The config for that client on nordiacpis.curity.io is with `client-1` as audience.

Comment (travisspencer) [reply settings.json]: but it won't work for default setups -- like demos, sales calls, etc. `audience` can be an array, maybe we should set the default to be `["client-1", "client-one]`.

File: client.py (hunk 2 above)
Comment (travisspencer) [reply client.py]: This is fine. I just missed the method before in the GH view of the code.

**Comment by travisspencer** [comment]
What are we doing with this? Shall we decline it?

**Comment by daniellindau** [comment]
I think so. Too much work to keep supporting both 2.7 and 3.0

**Pull request closed without merging**